test/unit/cli.test.js

```diff
@@ -1,5 +1,6 @@
-describe('cli parser', function () {
-    var cli = require('../../bin/newman');
+describe.skip('cli parser', function () {
```

Comment:
These tests have to be ported to the CLI tests.

Reply:
no @kunagpal - these are cli unit tests